### PR TITLE
Updated framework How to

### PR DESCRIPTION
**Pull request by NupurSancheti**
@Sunita: Generated the PR to get the document reviewed. I will inform after the technical review to merge in the QA.

### CODE REVIEW

**Comment by lakhanmandloi** [comment]
Hello @NupurSancheti,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://312.qa.docs.sunbird.org/1.11/ .<br/> **Note:** Only _1.11_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://312.qa.docs.sunbird.org/1.11/developer-docs/how-to-guide/how_to_create_framework_in_sunbird/ <br />  <br/> **Modified sections are -** <br />

**Comment by kochhar** [comment]
Sure. Support team is creating forms using Jira service desk. Please
coordinate with Vatsa and Vinod to setup the form and fields needed.


Thanks,
Shailesh

**Comment by lakhanmandloi** [comment]
Hello @NupurSancheti,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://312.qa.docs.sunbird.org/1.11/ .<br/> **Note:** Only _1.11_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://312.qa.docs.sunbird.org/1.11/developer-docs/how-to-guide/how_to_create_framework_in_sunbird/ <br /> - http://312.qa.docs.sunbird.org/1.11/developer-docs/how-to-guide/how_to_seeddata_in_framework/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/how-to-guide/how_to_create_framework_in_sunbird.md [hunk 1]
```diff
@@ -137,55 +238,76 @@ The sample values provided in the request body are indicative. The API describes
             }
     }
 
+**Description of Parameters**
+
+Node_ID: The name of the framework as provided by the organization
```
Comment (amitpriyadarshi) [inline developer-docs/how-to-guide/how_to_create_framework_in_sunbird.md]: id of the framework category

**Comment by lakhanmandloi** [comment]
Hello @NupurSancheti,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://312.qa.docs.sunbird.org/1.11/ .<br/> **Note:** Only _1.11_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://312.qa.docs.sunbird.org/1.11/developer-docs/how-to-guide/how_to_create_framework_in_sunbird/ <br /> - http://312.qa.docs.sunbird.org/1.11/developer-docs/how-to-guide/how_to_seeddata_in_framework/ <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @NupurSancheti,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://312.qa.docs.sunbird.org/1.11/ .<br/> **Note:** Only _1.11_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://312.qa.docs.sunbird.org/1.11/developer-docs/how-to-guide/how_to_create_framework_in_sunbird/ <br /> - http://312.qa.docs.sunbird.org/1.11/developer-docs/how-to-guide/how_to_seeddata_in_framework/ <br />  <br/> **Modified sections are -** <br />

**Pull request merged**